src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,34 @@
+package racingcar
+
+class Cars {
+    private val cars = mutableListOf<Car>()
+
+    fun add(car: Car) {
+        cars.add(car)
+    }
+
+    fun getSize(): Int {
+        return cars.size
+    }
```

Comment:
kotlin 스럽게 표현한다면 아래와 같이 표현할 수 있습니다.

다른곳도 찾아서 적용해보시면 좋을것 같습니다.

```suggestion
    val size: Int
        get() = cars.size
```

Reply:
오 신기합니다!